client/verta/verta/runtime.py

```diff
@@ -16,10 +16,8 @@
 
 def _get_thread_logs() -> Dict[str, Any]:
     """
-    Return the current thread-local logs or initialize an empty dict.
+    Return the 'logs' attribute of the thread-local variable.
     """
-    if not hasattr(_THREAD, 'logs'):
-        _set_thread_logs(dict())
```

Comment:
We still need this `if`, maybe just to `return {}` or `return None`.

Otherwise PyMS will encounter an error if nothing was logged

```python
from verta import runtime

with runtime.context() as ctx:
    pass
ctx.logs()
```

```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In [1], line 6
      3 with runtime.context() as ctx:
      4     pass
----> 6 ctx.logs()

File ~/Documents/modeldb/client/verta/verta/runtime.py:242, in context.logs(self)
    231 def logs(self) -> Dict[str, Any]:
    232     """
    233     Return the currently stored, thread-local logs from inside this
    234     context manager. If called after exiting the context manager, the
   (...)
    240         Dictionary of logs collected within this context manager.
    241     """
--> 242     return  self._logs_dict or _get_thread_logs()

File ~/Documents/modeldb/client/verta/verta/runtime.py:21, in _get_thread_logs()
     17 def _get_thread_logs() -> Dict[str, Any]:
     18     """
     19     Return the 'logs' attribute of the thread-local variable.
     20     """
---> 21     return _THREAD.logs

AttributeError: '_thread._local' object has no attribute 'logs'
```

Reply:
☝️ That might also be a good test to add.

Reply:
Added this test.